pycytominer/cyto_utils/__init__.py

```diff
@@ -40,4 +40,14 @@
     aggregate_fields_count,
     aggregate_image_features,
 )
-from .collate import collate
+
+# Conditionally import collate function depending on if optional dependencies installed
```

Comment:
Partially about this line and more: depending on what you think about the other comment, to be extra thorough it might be a good idea to keep this comment (or something like it) around the function-local import so anyone looking at it understands the necessity of the import location and avoids what might look like "import organization cleanup work".

This brings up a different point about a fork in the road with test cases we have at the moment from source vs conda. To avoid this issue in the future ahead of a conda release should we try the very same test found in the conda recipe (i.e. `pip install pycytominer; import pycytominer`)? This might be considered ["shifting left on testing"](https://en.wikipedia.org/wiki/Shift-left_testing) here, detecting things earlier in the process with theoretical related benefits.

Reply:
I definitely like the shift left on testing approach. I think it would be good to define which tests require which optional dependencies and configure tests to run in minimal environments. The one concern I would have would be about the testing matrix getting unwieldly large, but it might also mean faster more parallelized testing...

Reply:
I had a hard time figuring out what kind of comment would be appropriate here based on your suggestion. We're not doing any other library imports in __init__.py so I'm not sure whether this a place someone would ever come looking for that type of information.

Reply:
Thanks @kenibrewer , I think the changes you've added look great for now. Besides the changes you added, generally I wonder if a short GitHub Actions test for the installation and import of Pycytominer might avoid potential issues with the conda package testing (in advance of recipe updates). This might be overdoing it but could help avoid issues in the future.